backend/sftp/file.go

```diff
@@ -281,7 +282,7 @@ func (f *File) Seek(offset int64, whence int) (int64, error) {
 	f.fileSystem.connTimerStop()
 	defer f.fileSystem.connTimerStart()
 
-	sftpfile, err := f.openFile(os.O_RDWR)
+	sftpfile, err := f.openFile(os.O_RDONLY)
```

Comment:
I should have asked this last time, but would you explain what calling openFile 2 times accomplishes?